dashboard/app/views/levels/editors/fields/_dsl_fields.html.haml

```diff
@@ -48,6 +48,40 @@
       The Lab2 implementation can host non-Lab2 sublevels, but only transitions
       between Lab2 levels will be done without a page reload.
 
+  - if @level.is_a?(Multi)
+    %h4 Tip: Allow Multiple Attempts
+    %p
+      Set the <code>allow_multiple_attempts</code> property to <code>false</code> in the DSL
+      to give students only a single opportunity to answer the question. By default, students
+      are allowed multiple attempts.
```

Comment:
Can we double-check this? I thought the default behavior was to not allow multiple attempts, but I might be misremembering

Reply:
Yes, confirming default is `false` in `multi.rb`.
https://github.com/code-dot-org/code-dot-org/blob/b6bb9aa436b5a27cea694a63e624eee9223bdb64/dashboard/app/models/levels/multi.rb#L39